source/module_rdmft/rdmft.h

```diff
@@ -0,0 +1,200 @@
+//==========================================================
+// Author: Jingang Han
+// DATE : 2024-03-11
+//==========================================================
+#ifndef RDMFT_H
+#define RDMFT_H
+
+
+#include "module_parameter/parameter.h"
+#include "module_hamilt_pw/hamilt_pwdft/global.h"
+#include "module_psi/psi.h"
+#include "module_base/matrix.h"
+
+#include "module_basis/module_ao/parallel_2d.h"
+#include "module_basis/module_ao/parallel_orbitals.h"
+#include "module_cell/unitcell.h"
+#include "module_hamilt_lcao/module_gint/gint_gamma.h"
+#include "module_hamilt_lcao/module_gint/gint_k.h"
+#include "module_basis/module_ao/ORB_read.h"
+#include "module_basis/module_nao/two_center_bundle.h"
+
+#include "module_hamilt_lcao/hamilt_lcaodft/operator_lcao/operator_lcao.h"
+#include "module_hamilt_lcao/module_hcontainer/hcontainer.h"
+#include "module_hamilt_lcao/hamilt_lcaodft/hs_matrix_k.hpp"
+
+#ifdef __EXX
+#include "module_ri/Exx_LRI.h"
+#include "module_ri/module_exx_symmetry/symmetry_rotation.h"
+// there are some operator reload to print data in different formats
+#include "module_ri/test_code/test_function.h"
+#endif
+
+#include <iostream>
+#include <type_traits>
+#include <complex>
+#include <vector>
+#include <iomanip>
+
+
+
+namespace rdmft
+{
+
+
+template <typename TK, typename TR>
+class RDMFT
+{
+
+  public:
+    RDMFT();
+    ~RDMFT();
+    
+    const Parallel_Orbitals* ParaV = nullptr;
+    Parallel_2D para_Eij;
+    elecstate::ElecState* pelec = nullptr;  // just to gain Ewald and this->pelec->pot
+    const K_Vectors* kv = nullptr; // update after ion step
+
+    int nk_total = 0;
+    int nbands_total;
+    int nspin = 1;
+    std::string XC_func_rdmft;
+    double alpha_power = 0.656; // 0.656 for soilds, 0.525 for dissociation of H2, 0.55~0.58 for HEG
+
+    // natrual occupation numbers and wavefunction
+    ModuleBase::matrix occ_number;
+    psi::Psi<TK> wfc;
```

Comment:
this is a good practice: once possible, name variable with physical name instead of some symbol.